spec/models/project_spec.rb

```diff
@@ -18,13 +18,48 @@
         end
 
         before do
+          mock_successful_api_conversion_project_response(urn: 12345)
           allow_any_instance_of(AcademiesApi::Client).to \
             receive(:get_establishment) { establishment_result }
         end
 
         it "is invalid" do
           expect(subject).to_not be_valid
-          subject.errors[:urn].include?("No establishment exists with that URN. Enter a valid URN.")
+          expect(subject.errors[:urn]).to include(I18n.t("activerecord.errors.models.project.no_establishment_found"))
+        end
+      end
+
+      context "when no conversion project with that URN exists in the API" do
+        let(:conversion_project_result) do
+          AcademiesApi::Client::Result.new(nil, AcademiesApi::Client::NotFoundError.new(I18n.t("academies_api.get_conversion_project.errors.not_found", urn: 12345)))
+        end
+
+        before do
+          mock_successful_api_establishment_response(urn: 12345)
+          allow_any_instance_of(AcademiesApi::Client).to \
+            receive(:get_conversion_project) { conversion_project_result }
+        end
+
+        it "is invalid" do
+          expect(subject).to_not be_valid
+          expect(subject.errors[:urn]).to include(I18n.t("activerecord.errors.models.project.no_conversion_project_found"))
+        end
+      end
+
+      context "when no conversion project with that URN exists in the API" do
```

Comment:
Copy paste error here. Should be "when the API returns multiple conversion projects" (or similar)